w_tilde/w_tilde_curvature.py

```diff
@@ -1,17 +1,22 @@
+from __future__ import annotations
+
 import numpy as np
 from os import path
 
+from jax import jit
+import jax.numpy as jnp
+
 from autoarray import numba_util
 
 import autolens as al
 
 
-@numba_util.jit()
+@jit
 def w_tilde_curvature_interferometer_from(
-    noise_map_real: np.ndarray,
-    uv_wavelengths: np.ndarray,
-    grid_radians_slim: np.ndarray,
-) -> np.ndarray:
+    noise_map_real: np.ndarray[tuple[int], np.float64],
+    uv_wavelengths: np.ndarray[tuple[int, int], np.float64],
+    grid_radians_slim: np.ndarray[tuple[int, int], np.float64],
+) -> np.ndarray[tuple[int, int], np.float64]:

```

Comment:
We're still supporting older versions of Python so could we use type annotations from the typing library?

e.g.
```python
from typing import Tuple

Tuple[int]

Reply:
Resolve in ac1ed27